public/app/features/explore/utils/supplementaryQueries.ts

```diff
@@ -84,9 +85,11 @@ const createFallbackLogVolumeProvider = (
         const bucketSize = exploreData.logsResult.bucketSize;
         const targetRefIds = queryTargets.map((query) => query.refId);
         const rowsByRefId = groupBy(exploreData.logsResult.rows, 'dataFrame.refId');
+        let allSeries: DataFrame[] = [];
```

Comment:
The aggregation for the fallback was emitting only the last value instead of aggregating them. I've added a new test `Creates single fallback result` that covers for it (it'd fail without these changes).